src/zenml/cli/user_management.py

```diff
@@ -191,26 +204,78 @@ def create_user(
     required=False,
     help="New user email.",
 )
+@click.option(
+    "--password",
+    "-p",
+    "updated_password",
+    type=str,
+    required=False,
+    help="New user password.",
+)
+@click.option(
+    "--admin",
+    "-a",
+    "make_admin",
+    is_flag=True,
+    required=False,
+    default=None,
+    help="Whether the user should be an admin.",
+)
+@click.option(
+    "--user",
+    "-u",
+    "make_user",
+    is_flag=True,
+    required=False,
+    default=None,
+    help="Whether the user should be a regular user.",
+)
 def update_user(
     user_name_or_id: str,
     updated_name: Optional[str] = None,
     updated_full_name: Optional[str] = None,
     updated_email: Optional[str] = None,
+    updated_password: Optional[str] = None,
+    make_admin: Optional[bool] = None,
+    make_user: Optional[bool] = None,
 ) -> None:
-    """Create a new user.
+    """Update an existing user.
 
     Args:
         user_name_or_id: The name of the user to create.
         updated_name: The name of the user to create.
         updated_full_name: The name of the user to create.
         updated_email: The name of the user to create.
+        updated_password: The name of the user to create.
+        make_admin: Whether the user should be an admin.
+        make_user: Whether the user should be a regular user.
     """
+    if make_admin is not None and make_user is not None:
+        cli_utils.error(
+            "Cannot set both --admin and --user flags as mutually exclusive."
+        )
     try:
+        current_user = Client().get_user(
+            user_name_or_id, allow_name_prefix_match=False
+        )
+        if current_user.is_admin and make_user:
+            confirmation = cli_utils.confirmation(
+                f"Currently user `{current_user.name}` is an admin, are you sure you to make they regular user?"
+            )
+            if not confirmation:
+                cli_utils.declare("User update canceled.")
+                return
         Client().update_user(
             name_id_or_prefix=user_name_or_id,
             updated_name=updated_name,
             updated_full_name=updated_full_name,
             updated_email=updated_email,
+            updated_password=updated_password,
+            updated_is_admin=True
+            if make_admin is True
+            else False
+            if make_user is True
+            else None,
```

Comment:
```suggestion
            updated_is_admin = make_admin or (make_user and not make_admin)
```

Maybe a bit clearer? I think the logic is the same? Feel free to ignore it; just had to stare at the conditionals a bit to understand.

Reply:
A bit matter of taste, I'd say... The proposed fix is harder for me to understand than the initial statement.
I'll break it into a if/elif logic for readability, once CI completes.

Reply:
It will be
```python
        updated_is_admin = None
        if make_admin is True:
            updated_is_admin = True
        elif make_user is True:
            updated_is_admin = False
```
Ok? @strickvl

Reply:
LGTM